src/plugins/share/public/components/url_panel_content.tsx

```diff
@@ -404,17 +402,26 @@ export class UrlPanelContent extends Component<UrlPanelContentProps, State> {
   };
 
   private renderExportUrlAsOptions = () => {
+    const snapshotLabel = (
+      <FormattedMessage id="share.urlPanel.snapshotLabel" defaultMessage="Snapshot" />
+    );
     return [
       {
         id: ExportUrlAsType.EXPORT_URL_AS_SNAPSHOT,
-        label: this.renderWithIconTip(
-          <FormattedMessage id="share.urlPanel.snapshotLabel" defaultMessage="Snapshot" />,
-          <FormattedMessage
-            id="share.urlPanel.snapshotDescription"
-            defaultMessage="Snapshot URLs encode the current state of the {objectType} in the URL itself.
+        label: (
+          <>
+            {this.props.snapshotShareWarning
+              ? this.renderWithWarning(snapshotLabel, this.props.snapshotShareWarning)
+              : this.renderWithIconTip(
```

Comment:
This logic ensures that only a single icon tooltip is shown at one time, with the error/warning state taking precedence - i.e. if there is a snapshot share warning, then only the warning icon will be shown:

However, if there is no warning, then the info icon will be shown:

Reply:
After feedback from [this](https://github.com/elastic/kibana/pull/141985#discussion_r984057229) discussion, this warning state has been modified to look like this:

![Oct-13-2022 11-12-58](https://user-images.githubusercontent.com/8698078/195662519-c635c08c-46bc-420a-b759-6a376b17447c.gif)